src/watchers/chat.js

```diff
@@ -1,73 +1,74 @@
-const $ = require('jquery');
-const twitch = require('../utils/twitch');
-const domObserver = require('../observers/dom');
+import $ from 'jquery';
+import twitch from '../utils/twitch.js';
+import domObserver from '../observers/dom.js';
 
-const PATCHED_SYMBOL = Symbol();
+const PATCHED_SYMBOL = Symbol('patched symbol');
 
 let twitchHandleMessage;
 let watcher;
 
 function bttvHandleMessage(message) {
-    if (message && typeof message.type === 'number') {
-        let isPrevented = false;
-        watcher.emit('chat.message.handler', {
-            message,
-            preventDefault: () => {
-                isPrevented = true;
-            }
-        });
-        if (isPrevented) return;
-    }
-
-    return twitchHandleMessage.call(this, ...arguments);
+  if (message && typeof message.type === 'number') {
+    let isPrevented = false;
+    watcher.emit('chat.message.handler', {
+      message,
+      preventDefault: () => {
+        isPrevented = true;
+      },
+    });
+    if (isPrevented) return false;
+  }
+
+  return twitchHandleMessage.call(this, message);
```

Comment:
this is intentionally using arguments since twitch could break this at any point. for forwards compatibility, we could use spreading on the function declaration, and spreading here